docs/intro/tutorial05.txt

```diff
@@ -489,7 +489,6 @@ class:
             If no questions exist, an appropriate message is displayed.
             """
             response = self.client.get(reverse('polls:index'))
-            self.assertEqual(response.status_code, 200)
```

Comment:
I think we should leave it even if it's redundant. This is a part of tutorials so it's better to be explicit. Moreover it matches a description from [The Django test client section](https://docs.djangoproject.com/en/3.1/intro/tutorial05/#the-django-test-client) that checks `status_code` and then `content`.